tests/countries/test_turks_and_caicos_islands.py

```
@@ -0,0 +1,171 @@
+    def setUpClass(cls):
+        super().setUpClass(TurksAndCaicosIslands, years=range(2020, 2025))
+
+    def test_country_aliases(self):
+        self.assertAliases(TurksAndCaicosIslands, TC, TCA)
+
+    def test_no_checks(self):
+        self.assertNoChecks(TurksAndCaicosIslands)
+
+    def test_commonwealth_day(self):
+        self.assertHolidayName(
+            "Commonwealth Day", 
```

Ruff (0.8.2)

Trailing whitespace

Remove trailing whitespace

(W291)